tests/gdx-tests-android/AndroidManifest.xml

```diff
@@ -4,7 +4,7 @@
       android:versionCode="1"
       android:versionName="1.0"
       android:installLocation="preferExternal">
-	<uses-sdk android:minSdkVersion="8" android:targetSdkVersion="19"/>
+	<uses-sdk android:minSdkVersion="9" android:targetSdkVersion="19"/>
```

Comment:
The support libraries now require a minSdkVersion version of 9. The other option is to override each and every one of them in the manifest.